modules/common/certificates/helm/certificates/templates/certificates.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: {{ Values.namespace }}-certificate
+  namespace: istio-system
+spec:
+  acme:
+    config:
+    - domains:
+      - {{ Values.cluster_domain }}
+      - *.{{ Values.cluster_domain }}
+      dns01:
+        provider: route53-liatrio
```

Comment:
I'm introducing an output in the istio module which will have this provider name so it's not hard-coded

Reply:
Is there a branch youre working on @jphayes ?